### DISCUSSION TITLE
i18n support ?

### DISCUSSION THREAD

**Discussion by kpsjocelyn** [docusealco/docuseal#263] [Q&A]
Hello,
Is there any options to contribute to translating of DocuSeal on different languages ?

**Comment by omohokcoj** [comment]
Hi @kpsjocelyn which langauge are you looking for?

**Reply by kpsjocelyn** [reply]
Hi,
French language for all the application, I'm French, I can write all the translation

**Reply by omohokcoj** [reply]
@kpsjocelyn French is available for the signing form https://www.docuseal.co/resources/languages-support
The internal admin UI is currently only in English (we do not accept translations for the admin UI at the moment but eventualyl it will be translated as well)

**Reply by ernestswagner** [reply]
Hi, I would also like to ask about translations and possible participation in doing one. I like DocuSeal a lot. Right now I am running some test with self hosting, but not excluding the possibility going commercial in few months. However, I need the Polish version at least for user UI. I don't see it in the list. I can do it if you want. Please LMK what is the possibility to contribute and if not, then is there a way I configure an alternative plan version by myself by modifying some file?

**Reply by mv-si** [reply]
It would be amazing if you added a way to translate the admin UI. I am also looking for a French version.

**Comment by omohokcoj** [comment]
For translation corrections/contribution see https://github.com/docusealco/docuseal-i18n
